.travis.yml

```diff
@@ -13,6 +13,7 @@ install:
   - ./hack/verify-gofmt.sh
   - ./hack/verify-boilerplate.sh
   - ./hack/verify-description.sh
+  - ./hack/verify-gendocs.sh
```

Comment:
`hack/verify-gendocs.sh` is already being run under the `script:` section. Fine to move it here, but delete from line 24 so we don't run it twice.